aeon/schema/social.py

```diff
@@ -0,0 +1,94 @@
+"""Readers for data relevant to Social experiments."""
+
+import json
+from pathlib import Path
+
+import pandas as pd
+
+import aeon.io.reader as _reader
+
+
+class Pose(_reader.Harp):
+    """Reader for Harp-binarized tracking data given a model that outputs id, parts, and likelihoods."""
+    def __init__(self, pattern):
+        self.pattern = pattern
+        self.extension = "bin"
+
+    def read(self, file):
+        """
+        Reads data from the Harp-binarized tracking file
+        """
+        # Get config file from `file`, then bodyparts from config file.
+        ceph_proc_dir = "/ceph/aeon/aeon/data/processed"
+        model_dir = file.stem.replace("_", "/")
+        config_file_dir = Path(ceph_proc_dir + "/" + model_dir)
+        assert config_file_dir.exists(), f"Cannot find model dir {config_file_dir}"
+        config_file = get_config_file(config_file_dir)
+        parts = self.get_bodyparts(config_file)
+        if parts is None:
+            raise ValueError(f"Cannot find bodyparts in {config_file}.")
+        # With bodyparts, set columns as set by Bonsai, and read data in default format.
```

Comment:
Minor meta-comment, but I would generalize this comment to the Harp format, e.g.:

`# Using bodyparts, assign column names to the Harp raw register values, and read data in default format.`